### DISCUSSION TITLE
What is the armbian source for the NanoPi Fire3 Image?

### DISCUSSION THREAD

**Discussion by port19x** [MichaIng/DietPi#6573] [Q&A] [answered]
I can't find it on the armbian website despite searching for quite a bit.
Perhaps they removed the upstream source?
I'd like to use armbian for the nodes, as I don't benefit from the UX improvements of dietpi, due to me automating setup with ansible

**Comment by MichaIng** [comment, marked as answer]
Indeed, Armbian dropped support for NanoPi Fire3 (as well as M3/T3) a long time ago, since they do not work with mainline Linux and rebasing the vendor sources onto newer kernel versions was basically seen as impossible.

Old kernel packages can still be found here: https://apt.armbian.com/pool/main/l/linux-4.14.180-s5p6818/
We host the bootloader package on our own server.

There is still a config for the Armbian build system, from where you can derive kernel and U-Boot sources: https://github.com/armbian/build/blob/main/config/boards/nanopifire3.eos

We still provide DietPi images for those SBCs, but will stop doing so once we drop support Debian Bullseye, probably earlier, since this old Linux 4.14 does not have all features required to run recent Debian versions.

**Reply by port19x** [reply]
Dang, seems like that cluster was an even worse investment than I thought

**Reply by MichaIng** [reply]
Sadly yes. It is a pain that FriendlyELEC and/or Samsung did never attempt to upstream support for this SBC/SoC. And they obviously made it sufficiently hard for the community to add/backwards engineer it. The octa-core is so nicely powerful, if you run applications which can make use of 8 cores. But with Linux 4.14 (the original kernel was even 4.4) it is not really usable anymore with modern distros/software.

I hate the fact that such SBCs are still sold. No idea if there is any chance, but it cannot hurt to try returning/refunding the cluster, given the reason that the kernel/software is critically/unusably outdated for any serious project and no alternatives (distros) are available, and no warning about the fact on any shop/product/support page. A change about this vendor practice can only come from customers.
